src/pages/Opportunities.jsx

```diff
@@ -0,0 +1,229 @@
+import React, { useState } from "react";
+import { ChevronLeftIcon, ChevronRightIcon } from "@heroicons/react/20/solid";
+
+function Opportunities() {
+  const [activeCategories, setActiveCategories] = useState([]);
+
+  const buttons = [
+    { text: "All Issues" },
+    { text: "Category 1" },
+    { text: "Category 2" },
+    { text: "Category 3" },
+  ];
+
+  const handleButtonClick = (buttonText) => {
+    if (buttonText === "All Issues") {
+      setActiveCategories([]);
+    } else {
+      setActiveCategories((prevCategories) =>
+        prevCategories.includes(buttonText)
+          ? prevCategories.filter((category) => category !== buttonText)
+          : [...prevCategories, buttonText]
+      );
+    }
+  };
+
+  const opportunitiesList = [
```

Comment:
For organization, move this array to a `data.js` file (maybe in the util folder), export from there into this file.

Reply:
@justgogame